Dockerfile.goreleaser

```diff
@@ -7,6 +7,7 @@ RUN ["/busybox/ln", "-s", "/busybox/cp", "/bin/cp"]
 COPY olm /bin/olm
 COPY catalog /bin/catalog
 COPY package-server /bin/package-server
+COPY copy-content /bin/copy-content
```

Comment:
Do I understand correctly - this is what ends up in the image we push to quay on release?

Reply:
That's right

Reply:
We've been shipping without it =S

Reply:
Downstream has been fine because we don't use this makefile or image for building/releasing